### ISSUE TITLE
pglm data sets

### ISSUE THREAD

**Issue by Nosferican** [johnmyleswhite/RDatasets.jl#47]
[pglm](https://CRAN.R-project.org/package=pglm) has four data sets that could be added:
- Fairness
- HealthIns
- PatsRD
- Unions

**Comment by Nosferican** [comment]
I know there was some issue with who was maintaining the package since @johnmyleswhite wasn't very active these days (issue raised during updating GLM). I don't know who is maintaining it. If there is no one maintaining it then I guess I could learn the internals of the package. If there is someone actively maintaining the packages and is familiar with the internals I imagined the process to add new datasets from CRAN packages to be quite low in terms of time and effort (I assume is the same process as with all the others).

**Comment by randyzwitch** [comment]
This package has never been in disrepair, as myself and several other people have had merge access for quite some time.

While it _may_ be easy for one of us to add a dataset (hard to say until you actually get in and do it), ultimately it will get done the fastest by the person who actually wants to use it. RDatasets was never meant to be an exact mirror, but rather, a means for having _some_ example datasets available. In that sense, the package has achieved that goal and why you don't see a lot of movement here.

**Comment by Nosferican** [comment]
Could you then tell me or add some quick Contribute/docs to add new datasets? CSV.gz the data set checkout those in `data` add the names in the lines at `src` and profit?

**Comment by randyzwitch** [comment]
Yes, you generally add the file into its directory. The test suite shows all of the things that need to pass to make sure the dataset loads properly.
